tests/installation/ipxe_install.pm

```diff
@@ -108,7 +118,7 @@ echo ++++++++++++ openQA ipxe boot ++++++++++++
 echo +    Host: $host
 echo ++++++++++++++++++++++++++++++++++++++++++
 
-kernel $kernel $install $cmdline_extra
+kernel $kernel $install $cmdline_extra_para
```

Comment:
What's the purpose of this rename?  If you didn't change the scope of $cmdline_extra below you should not need this.